index.d.ts

```diff
@@ -45,6 +45,11 @@ declare class BeeQueue {
   close(timeout?: number | null): Promise<void>;
   close(timeout: number | undefined | null, cb: () => void): void;
 
+  isRunning(): boolean;
+
+  ready(): Promise<this>;
+  ready(cb: () => void): void;
```

Comment:
If I'm not mistaken, this still returns a `Promise` - maybe these two definitions should just get merged to have an optional `cb`?
```suggestion
  ready(cb: () => void): Promise<this>;
```

Reply:
Thanks -- yes, a promise is returned.

Regarding merging the declarations: I'm new to the creating of `*.d.ts` files.  However, this multi-line pattern appears throughout the `@tkypes/node/*.d.ts` files, e.g. in `fs.d.ts` which has a lot of legacy support...